content/documentation.rst

```diff
@@ -431,3 +450,18 @@ page.
 
 Known Issues will include a reference to the internal Trac ID which provided
 further details about that issues
+
+Mock ups and Designs
+====================
+
+If a change involves a design or content addition (such as an image carousel in JS), it is a good idea to write/mock up the content first before doing any coding.  In this way, you can check to see what type of code work should be done to best communicate the content.
+
+Please go to the 'design' repository for sample images and screenshots to use and add your own samples.
+
+If raw HTML needs to be used, aim to use directives such as:
```

Comment:
Maybe.

Instead of using raw HTML try to see if you can get the same results by creating custom directives.
This will allow transfroming these directives into HTML, but also into PDF or other formats... while raw HTML will be ingored in PDF or other non-HTML formats.

Below is an example of a directive used in the website to insert a `call for action` button... which is seen/hhandled as a standardiezed component